src/walletExplorer.ts

```diff
@@ -0,0 +1,254 @@
+            (this.label || path.basename(this.jsonFile));
+        const result = new vscode.TreeItem(treeItemLabel, (this.index !== undefined) ? vscode.TreeItemCollapsibleState.None : vscode.TreeItemCollapsibleState.Expanded);
+        result.description = (this.index !== undefined) ? '' : this.jsonFileName;
+        result.iconPath = (this.index !== undefined) ? vscode.ThemeIcon.File : vscode.ThemeIcon.Folder;
+        result.tooltip = (this.index !== undefined) ? '' : 'Wallet loaded from: ' + this.jsonFile;
+        result.contextValue = (this.index !== undefined) ? '' : 'cancreate';
+        if (this.index !== undefined) {
+            result.command = {
+                title: 'Open wallet',
+                command: 'neo-visual-devtracker.openWallet',
+                arguments: [this.jsonFile],
+            };
+        }
+        return result;
+    }
+
+    public async createAccount() {
+        if (this.jsonFile) {
+            const jsonFileContents = fs.readFileSync(this.jsonFile, { encoding: 'utf8' });
+            const contents = JSON.parse(jsonFileContents);
+            const parsedWallet = new wallet.Wallet(contents);
+            const passphrase = await vscode.window.showInputBox({
+                prompt: 'Enter the passphrase for ' + path.basename(this.jsonFile),
+                password: true,
+                ignoreFocusOut: true,
+            });
+            if (passphrase) {
+                let decrypted = false;
+                try {
+                    decrypted = (await parsedWallet.decryptAll(passphrase)).reduce((a, b) => a && b, true);
+                } catch (e) {
+                    console.error('Wallet decryption error', this.jsonFile, e);
+                }
+                if (decrypted) {
+                    const accountName = await vscode.window.showInputBox({
+                        prompt: 'Enter a name for the new account',
+                        ignoreFocusOut: true,
+                    });
+                    if (accountName) {
+                        const account = new wallet.Account(wallet.generatePrivateKey());
+                        account.label = accountName;
+                        parsedWallet.addAccount(account);
+                        if (await parsedWallet.encryptAll(passphrase)) {
+                            fs.writeFileSync(this.jsonFile, JSON.stringify(parsedWallet.export(), undefined, 4));
+                        } else {
+                            vscode.window.showErrorMessage('The wallet file could not be encrypted using the supplied passphrase, the account was not added.', { modal: true });
+                        }
+                    }
+                } else {
+                    vscode.window.showErrorMessage('The passphrase supplied was incorrect.', { modal: true });
+                }
+            }
+        }
+    }
+}
+
+export class WalletExplorer implements vscode.TreeDataProvider<WalletTreeItemIdentifier> {
```

Comment:
I'm not sure a tree view is the best UI metaphor here, though honestly I can't think of something else. Maybe we should just be adding a "create account" item to teh main file explorer tree view if we detect that a given json file is a NEP-6 wallet + just have a "create wallet file" command w/o an explorer UI?

Reply:
When adding context menu items to the file explorer I only get the choice of matching based on the filename--I'd have to have the "Create account" option show up for all .json files not just NEP-6 wallets.  I think that would be confusing.

I'd like to do something like the "Add configuration..." button you see in the editor whenever you edit a launch.json file.  I can't find a way to do it though, I think the floating button in launch.json editors may be a special case built-in to vscode and not an actual extensibility point.  The closest I have managed to achieve is using a "Code Lens" to add a "Create account" link to the top of the editor window whenever you edit a NEP-6 wallet.